doc/whats_new.rst

```diff
@@ -62,6 +62,16 @@ Model Selection Enhancements and API Changes
     the corresponding parameter is not applicable. Additionally a list of all
     the parameter dicts are stored at ``results_['params']``.
 
+  - **Renaming of ``n_folds`` and ``n_iter`` to ``n_splits``**
+
+    Some parameter names have changed: 
```

Comment:
Perhaps...? I'm unconvinced that this series of comments markedly contributes to the quality of the text in terms of convention, clarity or information structure.